src/shared/lending/oracles/BaseOracle.sol

```diff
@@ -39,7 +39,7 @@ contract BaseOracle is AggregatorV3Interface {
 
     bool public isCheckPriceDeviation;
 
-    uint256 constant internal DECIMAL_DIFFERENCE = 1e6;
+    uint256 constant internal DECIMAL_DIFFERENCE = 1e10;
```

Comment:
can you pls explain this change?

Reply:
fixed in https://github.com/FortressFinance/fortress-contracts/pull/31/commits/2006f092208a39a38adbcb84d14c58c9108a395e